src/extension.ts

```diff
@@ -56,6 +56,15 @@ export async function activate(context: ExtensionContext): Promise<RExtension> {
         rWorkspace
     );
 
+    // creates a custom context value for the workspace view
+    // only shows view when session watcher is enabled
+    const sessionBool = workspace.getConfiguration('r')['sessionWatcher'];
```

Comment:
I think this could be moved down to https://github.com/Ikuyadeu/vscode-R/pull/487/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626R314-R315

where the session watcher setting is checked.

Reply:
Fix inc! Is there much of a difference between config() and workspace.getConfig? I hadn't seen that command in my googling